javascript/builtins/SharedArrayBuffer.json

```diff
@@ -17,9 +17,8 @@
               }
             ],
             "chrome_android": {
-              "version_added": "60",
-              "version_removed": "63",
-              "notes": "Chrome disabled SharedArrayBuffer on January 5, 2018 to help reduce the efficacy of <a href='https://www.chromium.org/Home/chromium-security/ssca'>speculative side-channel attacks</a>. This is intended as a temporary measure until other mitigations are in place."
+              "version_added": "89",
+              "notes": "SharedArrayBuffer are available on Android gated behind COOP/COEP"
```

Comment:
@ddbeck or @Elchi3 for notes review. I'm thinking we should link these terms to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy and https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy, or is there another article that explains what to do for COOP+COEP-guarded features?

Reply:
Linking to https://web.dev/coop-coep/ might be better?

Reply:
There is also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/Planned_changes but it is a bit outdated given these are not "planned changes" anymore. 

More pointers to docs that got written for this story are in https://github.com/mdn/sprints/issues/2219

Reply:
@Elchi3 is it OK to link to web.dev in the meantime?

Reply:
Sure. I filed https://github.com/mdn/content/issues/3489 for MDN.